### PR TITLE
[Tutorials] add tensorrt llm manual

### PR DESCRIPTION
**Pull request by lanking520**
## Description ##

Add TRTLLM conversion tutorial manual

### CODE REVIEW

File: serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+You need to manually login into the container to proceed for conversion
+
+```
+docker run -it --runtime=nvidia --gpus all \
+--shm-size 12g \
+deepjavalibrary/djl-serving:0.25.0-tensorrt-llm \
+/bin/bash
+```
+
+Then clone the TensorRT-LLM repository inside the container for manual conversion.
```
Comment (frankfliu) [inline serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md:47]: I think we should provide instruction on how to use our partition script to save the compiled model.

Comment (lanking520) [reply serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md:47]: We will eventually use partiton script to support it. But for now, we have to manually teach customers

File: serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+```
+# Build the Baichuan V1 13B model using a single GPU and FP16.
+python build.py --model_version v1_13b \
+                --model_dir baichuan-inc/Baichuan-13B-Chat \
+                --dtype float16 \
+                --use_gemm_plugin float16 \
+                --use_gpt_attention_plugin float16 \
+                --world_size 2 \
+                --output_dir baichuan_v1_13b/trt_engines/fp16/2-gpu/ \
+                --use_inflight_batching
```
Comment (rohithkrn) [inline serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md:90]: let's also add `--enable_context_fmha`, `--remove_input_padding` and `--parallel_build` as those args are always recommended.

File: serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
+# TensorRT-LLM manual compilation of models tutorial
+
+## Overview
+
+With LMI TensorRT-LLM container, you can run manual compilation of some models and quantization that LMI not supported for JIT and then load the model and run inference.
+
+The goal of this document is for the user to be able to:
+
+1. Convert any TensorRT-LLM supported model a model format that LMI can load and run inference.
```
Comment (rohithkrn) [inline serving/docs/lmi/tutorials/trtllm_manual_convert_tutorial.md:9]: missing "to",  `model to a model format`?

**Comment by rohithkrn** [comment]
Looks good overall, left minor comments.

**Comment by lanking520** [comment]
@rohithkrn addressed comments

**Review by rohithkrn — APPROVED** [review]

**Pull request merged**